firmware/hw_layer/debounce.h

```diff
@@ -28,13 +27,11 @@ class ButtonDebounce {
 private:
     int threshold;
     efitick_t timeLast;
-    brain_pin_e *pin;
-    brain_pin_e *active_pin;
+    brain_pin_e *pin = nullptr;
+    brain_pin_e *active_pin = nullptr;
     brain_pin_e oldPin;
```

Comment:
that's a bit weird why do we need "oldPin" permanently? please add code comment on why do we have to remember it - that's different from all other similar code where we get away without remembering old setting?

Reply:
I added that so when brain_pin_markUnused was called, it would call the pin that *was* set, but now that I think of it, we could just use active_pin. I'll change it.